src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -112,18 +123,21 @@ export class Conversations extends React.Component<Props, State> {
   }
 
   render() {
+    const { relay } = this.props
+    const showLoadingSpinner = relay && relay.hasMore() && this.state.fetchingNextPage
```

Comment:
How do we stub out relay environments in our tests? This guard is in place because `<Inbox />` is being tested further up the chain, and relay doesn't exist. 

I'm not 100% sure how its handled in Apollo, but in Redux (which uses a similar context-based) approach its easy to stub out entire trees just by wrapping the component in test inside of a `<Provider />`, ensuring deps don't fail in strange ways later.

Reply:
We're using jest mocks to stub out all of relay (https://github.com/artsy/emission/blob/master/__mocks__/react-relay.js) if there's a property or module missing in the stub, that's where it should go.

Reply:
Ahh perfect, nice and simple

Reply:
Fix ended up being `jest.mock` in the test in question, as it ended up being overly complex pass data deps down to `new Component(...)` from within the `__mocks__` folder.